ecoindex_cli/cli/app.py

```diff
@@ -102,14 +112,27 @@ def analyze(
         length=len(urls) * len(window_sizes),
         label="Processing",
     ) as progress:
+        err_found = False
         for url in urls:
             for w_s in window_sizes:
                 if url:
-                    results.append(
-                        asyncio.run(get_page_analysis(url=url.strip(), window_size=w_s))
-                    )
+                    try:
+                        results.append(
+                            asyncio.run(
+                                get_page_analysis(url=url.strip(), window_size=w_s)
+                            )
+                        )
+                    except Exception as e:
+                        err_found = True
+                        log.error("\n -- " + url + " --\n" + e.msg)
```

Comment:
```suggestion
                        error_found = True
                        log.error(" -- " + url + " -- " + e.msg)
```

I prefer not adding line break. But it is only my vision, so can be debated ;)